lib/fluent/plugin/out_amqp.rb

```diff
@@ -53,6 +53,7 @@ class AMQPOutput < Output
 
     config_section :buffer do
       config_set_default :@type, DEFAULT_BUFFER_TYPE
+      config_set_default :chunk_keys, ['tag']
```

Comment:
Does this ensure that the current behaviour where tag is defined is maintained, even though internally its effectivly renamed to `chunk_keys`?